trunk/src/app/srs_app_gb28181.cpp

```diff
@@ -1905,7 +1905,9 @@ srs_error_t SrsGbMuxer::write_h265_vps_sps_pps(uint32_t dts, uint32_t pts)
     }
 
     std::string sh;
-    if ((err = hevc_->mux_sequence_header(h265_vps_, h265_sps_, h265_pps_, sh)) != srs_success) {
+    std::vector<string> stl_VectorPPS;
```

Comment:
This line can be shortened here, and we should maintain consistent code style by avoiding the use of camel case for variable names as much as possible.
`std::vector<string> h265_pps = {h265_pps_};`